src/components/Alert/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ComponentType } from 'react';
+
+export type AlertProps = {
+  Icon: ComponentType;
+  dataTestId?: string;
+  alertHeader?: string;
+  errorMessage: string[];
+};
+
+const Alert = ({ Icon, dataTestId, alertHeader, errorMessage }: AlertProps): JSX.Element => {
+  return (
+    <div data-test-id={dataTestId} className="alert">
+      <div className="alert-content">
+        <div className="alert-icon" data-test-id="alert-icon">
+          <Icon />
+        </div>
+        <div className="alert-header-title">{alertHeader}</div>
```

Comment:
Let's use the correct HTML tag for this, like `h4` or `span` 😄 
The same goes for the `AlertWarning` component